### ISSUE TITLE
textlint-tester: remove babel dependencies

### ISSUE THREAD

**Issue by azu** [textlint/textlint#691]
https://github.com/textlint/textlint/blob/986f28d6435ad74f7b2745a5842926c06a849319/packages/textlint-tester/package.json#L47-L51

textlint-tester was rewritten by TypeScript.
babel dependencies is not necessary.

**Comment by heanzyzabala** [comment]
Hi. I would like to work on this one.

**Comment by azu** [comment]
@heanzyzabala Thanks! Welcome to PR!

If you can, `power-assert` also removable.
https://github.com/textlint/textlint/blob/986f28d6435ad74f7b2745a5842926c06a849319/packages/textlint-tester/package.json#L55
`power-assert` also is not necessary and it can be replaced with `assert`

```diff
- import assert from "power-assert"
+ import assert from "assert"
```

**Comment by heanzyzabala** [comment]
@azu Got it! Thanks

**Comment by azu** [comment]
closed by #692 

@heanzyzabala Thanks!
